src/components/Toolbar/__tests__/Toolbar.test.jsx

```diff
@@ -55,4 +56,42 @@ describe('Toolbar Component Tests', function() {
     expect(queryByText('Partners')).toBeInTheDocument();
     expect(queryByText('Sign Up')).not.toBeInTheDocument();
   });
+
+  test('Toolbar resized to width over medium screen breakpoint; Mobile Navigation Menu Closed', () => {
+    const { queryByLabelText, getByTestId } = setup();
+    window.innerWidth = MEDIUM_BREAKPOINT - 10;
+    window.dispatchEvent(new Event('resize'));
+
+    fireEvent.click(queryByLabelText('hamburger'));
+
+    expect(getByTestId('nav').getAttribute('class').includes('open')).toBeTruthy();
+
+    act(() => {
+      window.innerWidth = MEDIUM_BREAKPOINT + 10;
+      window.dispatchEvent(new Event('resize'));
+    });
+
+    expect(getByTestId('nav').getAttribute('class').includes('open')).toBeFalsy();
+  });
+
+  test('Trying to click a link with navbar closed', () => {
```

Comment:
this test doesn't really make sense to me, it only includes 'open' in the className once the hamburger has been clicked. This just seems to click links and the assertion has no correlation. I think the test after this is solid enough to cover the functionality and you can remove this test altogether.